src/quacc/recipes/emt/phonons.py

```diff
@@ -92,15 +98,16 @@ def phonon_flow(
         parameters=job_params,
         decorators=job_decorators,
     )
-    if run_relax:
+    if run_relax and not non_displaced_atoms:
```

Comment:
This is a little bit problematic, another option is to send both displaced and non_displaced and optimise them and separate them again...

Reply:
I see. I mean, I guess we can ask if we really even need a relaxation step beforehand? Presumably one could just call `relax_job()` before calling the flow and it would be exactly the same (although without a tight force tolerance).

If there's a desire to keep this though, then yeah I think the only route would be to relax `displaced_atoms+non_displaced_atoms` as a single `combined_atoms` and then pass in `combined_atoms[:len(displaced_atoms)]` etc. to `phonon_subflow`.